packages/stark-ui/src/modules/action-bar/components/action.intf.ts

```diff
@@ -2,6 +2,11 @@
  * Stark Action Base interface
  */
 export interface StarkActionBase {
+	/**
+	 * Color of the button (optional)
+	 */
+	buttonColor?: string;
```

Comment:
The "(optional)" string in the comment is not needed since it is already implicit in the Typescript code. Then, I think the type of this property should be more specific as @SuperITMan suggested, so it would be something like this:

```typescript
/**
 * Color of the action button. Default: "primary".
 */
buttonColor?: "primary" | "alert" | "accent" | "anyOtherStarkColor" | string;
```

In this case, the "string" type is still needed so the developer can give any other color.